gui_views/state.py

```diff
@@ -96,6 +101,18 @@ def addObjectToCatalog(self, object, object_name):
         """add the passed in imported object to the catalog"""
         self.mainWindow.addImportedObjectToCatalog(object, object_name)
 
+    def highlightIncomplete(self):
+        """color the object red if a parameter is not set"""
+        incomplete = False
+        for object in self.sym_objects.values():
+            for param in object.instance_params.values():
+                if param["Value"] == 'None' or param["Value"] == None:
```

Comment:
aren't some parameters optional so they'll be none and thus shouldn't be highlighted?

Reply:
We're currently highlighting all empty parameters in the attribute table so I thought we should be consistent with that